Documentation/FrameworkOverview.md

```diff
@@ -8,196 +8,219 @@ learning about new modules and finding more specific documentation.
-   finish. The event may include an `NSError` object that indicates what went
-   wrong. Errors must be handled specially – they are not included in the
-   stream's values.
+   finish. The event may include an signal specific `ErrorType` object that 
+   indicates what went wrong. If no error can happen, the `NoError` type can 
+   be specified. Errors must be handled specially – they are not included in 
+   the stream's values.
  * The **completed** event indicates that the signal finished successfully, and
    that no more values will be added to the stream. Completion must be handled
    specially – it is not included in the stream of values.
+ * The **interrupted** event indicates that the signal has terminated 
+   non-erroneous, yet unsuccessful e.g. when the corresponding request has been
+   cancelled before the signal could finish. Interruptions must be handeled 
+   specially - they are not included in the stream's values.
 
-The lifetime of a signal consists of any number of `next` events, followed by
-one `error` or `completed` event (but not both).
-
-### Subscription
 
-A **subscriber** is anything that is waiting or capable of waiting for events
-from a [signal](#signals). Within RAC, a subscriber is represented as any object
-that conforms to the [RACSubscriber][] protocol.
+Signals can be manipulated by appying [Operators][Operators] to them. 
+Typical operators to manipulate a single signal like `filter`, `map` and
+`reduce` are available as well as operators to manipulate multiple signals
+at once (`zip`).
+<!-- TODO: |> is a different kind of operator as signal operators. 
+How can this be clearified? -->
+The `|>` operator is used to apply operators to a signal. It can also be used
+to compose basic operators into more complex ones.
 
-A **subscription** is created through any call to
-[-subscribeNext:error:completed:][RACSignal], or one of the corresponding
-convenience methods. Technically, most [RACStream][] and
-[RACSignal][RACSignal+Operations] operators create subscriptions as well, but
-these intermediate subscriptions are usually an implementation detail.
 
-Subscriptions [retain their signals][Memory Management], and are automatically
-disposed of when the signal completes or errors. Subscriptions can also be
-[disposed of manually](#disposables).
+The lifetime of a signal consists of any number of `next` events, followed by
+one `error`, `completed` or `interrupted` event (but no combination of those).
 
-### Subjects
+### Pipes
 
-A **subject**, represented by the [RACSubject][] class, is a [signal](#signals)
+A **Pipe**, created by `Signal.pipe()`, is a [signal](#signals)
 that can be manually controlled.
 
-Subjects can be thought of as the "mutable" variant of a signal, much like
-`NSMutableArray` is for `NSArray`. They are extremely useful for bridging
-non-RAC code into the world of signals.
+The method returns a [Signal](#signals) and an [Observer](#observers). 
```

Comment:
May be better to call this an event sink for the signal (even though it has the same type as `Signal.Observer`)